src/TinyContainerBuilder.php

```diff
@@ -167,4 +167,43 @@ private function isAbsolute(string $path)
     {
         return (new Filesystem())->isAbsolutePath($path);
     }
+
+    public function excludeSourcePath(string $excludePath): ContainerBuilderInterface
+    {
+        if (!$this->isAbsolute($excludePath)) {
+            if (!isset($this->rootPath)) {
+                throw new \LogicException(
+                    \sprintf('When relative path is provided root should be set first. Privided: %s', $excludePath)
+                );
+            }
+            $excludePath = rtrim($this->rootPath, '/') . '/' . $excludePath;
+        }
+        if (!\file_exists($excludePath)) {
+            throw new \RuntimeException(\sprintf('Provided exclude path is not a valid pathname: %s', $excludePath));
+        }
```

Comment:
might be worth extracting this validation to its own function so it can be reused instead of copied in add/exclude

Reply:
I created the makePathAbsolute() method.